lib/matplotlib/axes/_axes.py

```diff
@@ -2725,7 +2725,7 @@ def pie(self, x, explode=None, labels=None, colors=None,
 
         Parameters
         ----------
-        x : array-like
+        x : 1D array-like
```

Comment:
Not necessarily needed. We generally assume array-like to be 1D if the parameter is semantically only 1D.

Reply:
edited back